data/src/main/kotlin/io/timemates/backend/data/users/datasource/PostgresqlUsersDataSource.kt

```diff
@@ -81,13 +95,13 @@ class PostgresqlUsersDataSource(private val database: Database) {
             get(UsersTable.USER_NAME),
             get(UsersTable.USER_EMAIL),
             get(UsersTable.USER_SHORT_DESC),
-            get(UsersTable.AVATAR_FILE_ID)
+            get(UsersTable.AVATAR_FILE_ID),
+            get(UsersTable.GRAVATAR_ID)
```

Comment:
add trailing comma; you can enable it in editor settings and reformat files you changed.